### PR TITLE
Modulesync 0.8.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/classes/confluence_service_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
       let(:params) do
         {
           javahome: '/opt/java',
-          version: '5.5.6',
+          version: '5.5.6'
```
Comment (igalic) [inline spec/classes/confluence_service_spec.rb]: what's up with this?

Comment (bastelfreak) [inline spec/classes/confluence_service_spec.rb]: there is no trailing comma allowed after the last element

Comment (igalic) [inline spec/classes/confluence_service_spec.rb]: why

Comment (bastelfreak) [inline spec/classes/confluence_service_spec.rb]: because rubocop dictates /o\

**Comment by igalic** [comment]
and so i bow before our rubocop overlord

**Pull request merged**